tests/v2/validation/provisioning/rke2/post_kdm_oob_release_test.go

```diff
@@ -0,0 +1,155 @@
+	}
+}
+
+func (k *KdmChecksTestSuite) TestProvisioningSingleNodeRKE2Clusters() {
+	require.GreaterOrEqual(k.T(), len(k.providers), 1)
+	require.GreaterOrEqual(k.T(), len(k.cnis), 1)
+
+	subSession := k.session.NewSession()
+	defer subSession.Cleanup()
+
+	client, err := k.client.WithSession(subSession)
+	require.NoError(k.T(), err)
+
+	kubeProvisioningClient, err := k.client.GetKubeAPIProvisioningClient()
+	require.NoError(k.T(), err)
+
+	providerName := k.providers[0]
+	provider := CreateProvider(providerName)
+	nodeRoles := k.nodesAndRoles
+
+	clusterNames := []string{}
+	clusterResps := []*v1.SteveAPIObject{}
+	k8sVersions := []string{}
+
+	// provisioning rke2 clusters using all released k8s versions
+	for _, k8sVersion := range k.rke2kubernetesVersions {
+
+		clusterName := namegen.AppendRandomString(provider.Name)
+
+		logrus.Info("provisioning " + k8sVersion + " cluster..")
+
+		cloudCredential, err := provider.CloudCredFunc(client)
+		require.NoError(k.T(), err)
+		generatedPoolName := fmt.Sprintf("nc-%s-pool1-", clusterName)
+		machinePoolConfig := provider.MachinePoolFunc(generatedPoolName, k.ns)
+		machineConfigResp, err := client.Steve.SteveType(provider.MachineConfigPoolResourceSteveType).Create(machinePoolConfig)
+		require.NoError(k.T(), err)
+		machinePools := machinepools.RKEMachinePoolSetup(nodeRoles, machineConfigResp)
+
+		cluster := clusters.NewK3SRKE2ClusterConfig(clusterName, k.ns, k.cnis[0], cloudCredential.ID, k8sVersion, machinePools)
+
+		clusterResp, err := clusters.CreateK3SRKE2Cluster(client, cluster)
+		require.NoError(k.T(), err)
+
+		clusterNames = append(clusterNames, clusterName)
+		clusterResps = append(clusterResps, clusterResp)
+		k8sVersions = append(k8sVersions, cluster.Spec.KubernetesVersion)
+	}
+
+	// checking for clusters to be up
+	k.checkClustersReady(client, kubeProvisioningClient, clusterResps, clusterNames, k8sVersions)
+}
+
+func (k *KdmChecksTestSuite) checkClustersReady(client *rancher.Client, kubeProvisioningClient *kubeProvisioning.Client, clusterResps []*v1.SteveAPIObject, clusterNames []string, k8sVersions []string) {
+	for i, clusterResp := range clusterResps {
+		logrus.Info("waiting for cluster ", clusterResp.Name, " with k8s version ", k.rke2kubernetesVersions[i], " to be up..")
+		result, err := kubeProvisioningClient.Clusters(k.ns).Watch(context.TODO(), metav1.ListOptions{
+			FieldSelector:  "metadata.name=" + clusterResp.Name,
+			TimeoutSeconds: &defaults.WatchTimeoutSeconds,
+		})
+		require.NoError(k.T(), err)
+		checkFunc := clusters.IsProvisioningClusterReady
+		err = wait.WatchWait(result, checkFunc)
+		require.NoError(k.T(), err)
+
+		assert.Equal(k.T(), clusterNames[i], clusterResp.Name)
+		assert.Equal(k.T(), k.rke2kubernetesVersions[i], k8sVersions[i])
+
+		clusterID, err := clusters.GetClusterIDByName(client, clusterResp.Name)
+		require.NoError(k.T(), err)
+
+		logrus.Infof("waiting for all pods to be up..")
```

Comment:
This logging can be removed as the function already does it.

Reply:
right, removed it.